input/blog/2020-11-25-config-bootstrap.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Extending the bootstrapper to parse cake.config
+category: How To's
+author: flcdrg
+---
+
+Cake makes a number of assumptions about files and directories in your project. If you're adding Cake to a brand new project then that isn't a problem, but what happens if you're adding Cake to a 'legacy' project that is already using some of these locations?
+
+This was a problem I faced with a project that was already using the `tools` directory, and at first glance I thought that [cake.config](https://github.com/cake-build/resources/blob/master/cake.config) would solve this, but it turns out this file is not referenced by the bootstrapper scripts.
```

Comment:
Maybe a link to https://cakebuild.net/docs/running-builds/configuration/set-configuration-values#configuration-file better?